src/app/containers/RadioSchedule/utilities/processRadioSchedule.js

```diff
@@ -20,6 +24,13 @@ export const getLink = (state, program, service) => {
   return state === 'live' ? `${url}/liveradio` : `${url}/${pid}`;
 };
 
+const logProgramError = error => {
+  logger.error({
```

Comment:
I think this was correct before, where you pass in the event name followed by an object containing the error - then it matches our sumologic log format

Reply:
Good point, I completely missed that.